.github/workflows/e2e.yml

```diff
@@ -49,11 +30,9 @@ jobs:
         run: pnpm exec playwright install --with-deps
 
       - name: Run Playwright tests
-        if: ${{ steps.deploy.outputs.origin-url }}
         run: pnpm run test:e2e
         env:
-          BASE_URL: ${{ steps.deploy.outputs.origin-url }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
 
       - uses: actions/upload-artifact@v4
```

Comment:
There is a warning that shows up `No files were found with the provided path: playwright-report/. No artifacts will be uploaded.`

We might want to look into this, or if this is a non-issue we could opt to silence this error through arguments to this action (though it's fine to stay as is too).

Reply:
Yeah, thanks, I just noticed that too. I'll look into it.